.github/CODEOWNERS

```diff
@@ -36,7 +36,7 @@
 # PRLabel: %Communication
 /sdk/communication/communication-identity/ @Azure/acs-identity-sdk 
 /sdk/communication/communication-chat/ @LuChen-Microsoft
-/sdk/communication/communication-phone-numbers/ @whisper6284 @miguhern @danielav7 @AlonsoMondal
+/sdk/communication/communication-phone-numbers/ @whisper6284 @miguhern @lucasrsant 
```

Comment:
👋 just FYI whoever is listed _first_ in the codeowners file will be the person assigned any nightly test failures. Not sure if you wanted to change your ordering based on that or if whisper6284 is still the right person - up to you folks!

Reply:
Thanks for the heads-up! Just sent out a commit addressing this 👍